lib/src/command/logout.dart

```diff
@@ -6,20 +6,37 @@ import 'dart:async';
 
 import '../command.dart';
 import '../oauth2.dart' as oauth2;
+import '../tokens.dart';
 
 /// Handles the `logout` pub command.
 class LogoutCommand extends PubCommand {
   @override
   String get name => 'logout';
   @override
-  String get description => 'Log out of pub.dartlang.org.';
+  String get description => 'Log out of pub.dev or any hosted server.';
```

Comment:
I think we need to update terminology throughout this PR.

`pub.dev` is a _hosted source_, so using `or` makes me think it's not.

Could we say `Log out of pub.dev or any third-party pub server.`.

Reply:
Great idea! Will change both login and logout commands for this.